demo/full/scripts/controllers/PlayerKnobsInfos.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import withModulesState from "../lib/withModulesState.jsx";
+// import AudioBitrateKnob from "./knobs/AudioBitrate.jsx";
+// import VideoBitrateKnob from "./knobs/VideoBitrate.jsx";
+// import LanguageKnob from "./knobs/AudioTrack.jsx";
+// import SubtitlesKnob from "./knobs/Subtitles.jsx";
+// import VideoTrack from "./knobs/VideoTrack.jsx";
+// import PlaybackRateKnob from "./knobs/SpeedKnob.jsx";
+
+const PlayerKnobsInfos = ({
+  shouldDisplay,
+  close,
+  // player,
+  // availableVideoTracks,
+  liveGap,
+  isContentLoaded,
+}) => {
+
+  if (!isContentLoaded) {
+    return null;
+  }
+
+  const className = "player-knobs" + (shouldDisplay ? " fade-in-out" : "");
+  const displayedLiveGap = Math.round(liveGap * 100) / 100;
+  return (
+    <div className={className}>
+      <div className="player-knobs-header">
+        <span className="player-knobs-title">Playback infos</span>
+        <span
+          className="player-knobs-close"
+          onClick={() => { close(); }}
+        >
+          {String.fromCharCode(0xf00d)}
+        </span>
+      </div>
+      <div className="player-knobs-content">
+        <p>{"Live Edge: " + displayedLiveGap}</p>
```

Comment:
So that's not really the `Live Edge`. More like the `distance to live` or `live gap` or `latence from live` or something